ghc-lib-gen/src/Ghclibgen.hs

```diff
@@ -42,19 +42,24 @@ cabalFileLibraries =
     , "compiler/ghc.cabal"
     ]
 
+-- These predicates are used so ubiquitously it makes sense to
+-- introduce some sugar.
+isGhcMaster, isGhc808, isGhcGreaterEqual810 :: GhcFlavor -> Bool
+isGhcMaster ghcFlavor = ghcFlavor == GhcMaster
+isGhc808 ghcFlavor = ghcFlavor `notElem` [ GhcMaster, Ghc8101]
```

Comment:
That’s a weird way to phrase this and will break as soon as we add a new version. Why not check for the versions you want instead of excluding the ones you don’t want?

Reply:
Yes. Improved occurrences to be explicit in this fashion.